problemsets-mat223/modules/module10-exercises.tex

```diff
@@ -1,17 +1,62 @@
 \begin{exercises}
 	\begin{problist}
-		\prob  Example problem with parts
+	    \prob Let $\mathcal S:\R^2\to\R^2$ be a transformation that doubles every vector.
+	    
+	    Find the corresponding matrix $M_{\mathcal S}$.
+	    
+	    \prob Let $\mathcal R:\R^2\to\R^2$ be rotation clockwise by $135^\circ$.
+	    
+	    Find the corresponding matrix $M_{\mathcal R}$.
+	    
+	    \prob Let $\mathcal P:\R^2\to\R^2$ be projection onto the $y$-axis.
+	    
+	    Find the corresponding matrix $M_{\mathcal P}$.
+	    \prob Let $\mathcal F:\R^2\to\R^2$ be a transformation reflecting over the line $y=x$.
+	    
+	    Find the corresponding matrix $M_{\mathcal F}$.
+	    
+	    \prob Try to decomposite the following linear transformations.  
+	    \begin{enumerate}
+	        \item Let $\mathcal U:\R^2\to\R^2$ be the matrix transformation given by $M=\mat{0&0\\-\sqrt{2}/2 & -\sqrt{2}/2}$.
+	        
+	         Let $\mathcal P:\R^2\to\R^2$ be projection onto the $y$-axis.
+	    
+	         Let $\mathcal R:\R^2\to\R^2$ be rotation clockwise by $135^\circ$.
```

Comment:
Why are these `Let` statements each in their own paragraph?

Reply:
You said the sentence is too long? What should I do with them then

Reply:
It's too long as code, not as English.

Reply:
I see ^-^